e2e/migrations/docker-run.sh

```diff
@@ -1,7 +1,16 @@
 CURRENT_DIRECTORY=`pwd`
 ROOT_DIRECTORY="$( dirname "$(dirname "$CURRENT_DIRECTORY")")"
 
-docker run --privileged -v \
+testOne= docker run --privileged -v \
 $ROOT_DIRECTORY:/gopkg/src/github.com/hashicorp/nomad \
 -it nomad-e2e /bin/bash \
--c "cd gopkg/src/github.com/hashicorp/nomad/e2e/migrations && go test -integration"
+-c "cd gopkg/src/github.com/hashicorp/nomad/e2e/migrations && go test --run \
+TestJobMigrations -integration"
+
+echo $testOne
```

Comment:
Are you not running test one anymore?

Reply:
This was just for printing test output, but is unnecessary. Removed.